packages/core-typings/src/IStats.ts

```diff
@@ -133,10 +133,10 @@ export interface IStats {
 		modules: string[];
 		tags: string[];
 		seatRequests: number;
-		livechatTags: number;
-		cannedResponses: number;
-		priorities: number;
-		businessUnits: number;
+		livechatTags?: number;
```

Comment:
Hey @cauefcr , you worked with stats collector recently. Do you think changing these props to be possibly undefined would affect how they are collected?